procurement_analytic/views/procurement_analytic.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2015 Carlos Dauden <carlos.dauden@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<openerp>
+<data>
+
+<record id="procurement_form_view_analiytic" model="ir.ui.view">
+    <field name="name">procurement_purchase_analytic.procurement.order.form</field>
+    <field name="model">procurement.order</field>
+    <field name="inherit_id" ref="procurement.procurement_form_view"/>
+    <field name="arch" type="xml">
+        <field name="group_id" position="after">
+            <field name="account_analytic_id" groups="purchase.group_analytic_accounting"/>
```

Comment:
Yes, this has been already discussed with @carlosdauden, and will be changed next week.